textile/features.textile

```diff
@@ -1481,9 +1481,13 @@ h4. Message
 *** @(TM7c1)@ @SummaryClientIdList@ is an object containing:
 **** @(TM7c1a)@ @total@ integer
 **** @(TM7c1b)@ @clientIds@ array of strings
+**** @(TM7c1c)@ @clipped@ boolean (must be forced to false if not set over the wire)
```

Comment:
Nitpick for consistency - a similar wording to TM2*, in form of "If not set on the wire, the SDK must ... "